src/test/java/com/atlassian/bitbucket/jenkins/internal/scm/BitbucketSCMDescriptorTest.java

```diff
@@ -348,6 +385,36 @@ public void testFillServerIdItemsValidId() {
         assertTrue(modelContains(model, serverConfigurationValid, true));
     }
 
+//    @Test
```

Comment:
Whats with this? Either delete or uncomment?

Reply:
Removed in latest PR.